src/Utils/gui/C3DTilesStyle.js

```diff
@@ -87,21 +87,21 @@ class C3DTilesStyle extends Widget {
                 /** @type {Map<HTMLElement, Function>} */
                 const colorFunctions = new Map();
 
-                const fillColorFunction = (c3DTileFeature) => {
+                const fillColorFunction = (styleContextProperties, styleContext) => {
```

Comment:
See above, you could have a way to not break the API style declaration. =)

Reply:
Yes I did not get that comment actually 
https://github.com/iTowns/itowns/blob/c862ca7376c195596c1111064f8189ccaad04348/src/Core/Style.js#L42